experimental/packages/opentelemetry-instrumentation-grpc/package.json

```diff
@@ -59,13 +59,11 @@
     "@protobuf-ts/runtime-rpc": "2.9.4",
     "@types/mocha": "10.0.10",
     "@types/node": "18.6.5",
-    "@types/semver": "7.5.8",
```

Comment:
Ah, you were doing this too. I noticed that some other packages had now-unused deps on semver and opened https://github.com/open-telemetry/opentelemetry-js/pull/5306
I should have read your patch first. :)

Reply:
Yep, there were some unused `semver` and some of them are only used by tests and can easily be replaced.